flathub-external-data-checker/entrypoint.sh

```diff
@@ -5,6 +5,7 @@ if [[ -z "$GITHUB_WORKSPACE" || -z "$GITHUB_REPOSITORY" ]]; then
     exit 1
 fi
 
+FEDC_OPTS=()
```

Comment:
This variable shouldn't be global, or we'll apply the same opts to all subsequent repos.
Need to figure out how to make it repo-specific.

Reply:
I think we can just define it at the start of detect_manifest()?

Reply:
Then how do we read it outside of the function body?

Reply:
I think if we echo FEDC_OPTS at the end of detect_manifest, we can pass it as a second output variable (along with manifest?

Alternatively, we could make a new function just for FEDC_OPTS opts, which returns FEDC_OPTS.

Reply:
I guess it would be simpler to just remove the function, move its body into the loop and replace `return 1` with `continue`

Reply:
indeed that's probably best, I've removed the function as suggested